lib/bambora/v1/profile_resource.rb

```diff
@@ -100,6 +100,54 @@ def get(customer_code:)
         client.get(path: "#{sub_path}/#{customer_code}", api_key: api_key)
       end
 
+      # Make a PUT Request.
+      #
+      # @example
+      #
+      #   client = Bambora::Rest::JSONClient(base_url: '...', api_key: '...', merchant_id: '...')
+      #   profiles = Bambora::V1::ProfileResource(client: client)
+      #
+      #   data = {
+      #     billing: {
+      #        name: "joh doe",
+      #        address_line1: "123 main st",
+      #        address_line2: "111",
+      #        city: "victoria",
+      #        province: "bc",
+      #        country: "ca",
+      #        postal_code: "V8T4M3",
+      #        phone_number: "25012312345",
+      #        email_address: "bill@smith.com"
+      #     },
+      #     card: {
+      #       name: 'Hup Podling',
+      #       number: '4030000010001234',
+      #       expiry_month: '12',
+      #       expiry_year: '23',
+      #       cvd: '123',
+      #     },
+      #   }
+      #
+      #   profiles.update(
+      #     path: 'v1/profiles',
+      #     body: data,
+      #     api_key: '...'
+      #   )
+      #   # => {
+      #   #      :code => 1,
+      #   #      :message => "Operation Successful",
+      #   #      :customer_code => "02355E2e58Bf488EAB4EaFAD7083dB6A",
+      #   #    }
+      #
+      # @param path [String] Indicating request path.
```

Comment:
The method signature and this documentation don't line up.

Reply:
I updated this so it matches up. Copy and paste has it's limitations lol